doc/src/install_guide/installation_guide.md

```diff
@@ -34,16 +17,19 @@ $ sudo dnf install gcc gcc-c++ make kernel-headers kernel-devel elfutils-libelf-
 
 Download the OPAE upstream kernel tree from github.
 ```console
-$ git clone https://github.com/OPAE/linux-dfl.git -b fpga-upstream-dev-5.8.0
+$ git clone https://github.com/OPAE/linux-dfl.git -b fpga-ofs-dev-5.15.lts
```

Comment:
fpga-ofs-dev-5.15.lts needs to change with **fpga-ofs-dev-5.15-lts**

Reply:
fixed